src/framework/preferences/js/PrefsEditor.js

```diff
@@ -371,38 +371,49 @@ var fluid_2_0 = fluid_2_0 || {};
     };
 
     /**
-     * Saves the current model and fires onSave
- */
-    fluid.prefs.prefsEditor.save = function (that) {
+     * Saves the value at the specified model path and fires onSave
+     * @param that: An fluid.prefs.prefsEditor instance
+     * @param modelPath: an array with segments of a model path or a string. If it's not given, save the entire model.
+     * @return the saved model
+     */
+    fluid.prefs.prefsEditor.save = function (that, modelPath) {
         if (!that.model) {  // Don't save a reset model
             return;
         }
 
-        var initialModel = that.initialModel,
-            userSelections = fluid.copy(that.model),
-            changedSelections = {};
+        var path = fluid.makeArray(modelPath).join("."),
+            currentModel = fluid.copy(that.model),
+            modelToSave = path === "" ? currentModel : that.getSettings();
```

Comment:
It seems strange that undefined, null, and "" will all cause the entire model to save. I'm thinking we either support a flag like saveEntireModel (boolean) or we allow for an array of paths such as [["path, "one"], "path.two"] where "" represents the entire model and undefined/null represent only the preferences. It seems like we only have a use case for the first example at this point, so it is probably enough. How does that sound?